Cargo.toml

```diff
@@ -116,17 +116,23 @@ reqwest = { version = "0.12.7" }
 ringbuffer = "0.15.0"
 rpassword = "7.3"
 rust-stream-ext-concurrent = "1.0.0"
-sea-orm = { version = "0.12.15", features = [
+scrypt = { version = "0.11.0", default-features = false }
+sea-orm = { version = "1.0.1", features = [
   "sqlx-sqlite",
   "with-chrono",
   "debug-print",
 ] }
-sea-orm-cli = { version = "0.12.15", features = ["codegen"] }
-sea-orm-migration = { version = "0.12.15", features = [
+sea-orm-cli = { version = "1.0.1", features = ["codegen"] }
+sea-orm-migration = { version = "1.0.1", features = [
   "sqlx-sqlite",
   "with-chrono",
 ] }
-scrypt = { version = "0.11.0", default-features = false }
+sea-query = { version = "0.31.0", default-features = false }
+sea-query-binder = { version = "0.5.0", default-features = false, features = [
+  "with-chrono",
+  "sqlx-sqlite",
+  "runtime-async-std-rustls",
```

Comment:
Should not this be selected rather by the crate itself depending on the runtime there?

Reply:
That's a testing setup, but likely needed also elsewhere.
Let's leave it like it is.